presentation/package.json

```diff
@@ -4,6 +4,13 @@
   "description": "Presentations of Cast projects made with sli.dev",
   "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "slidev --open",
+    "build": "slidev build --base /cast/",
```

Comment:
In relation to Lev's comment, would `/cast/` still be the base URL if we use CloudFlare Pages instead of GitHub's?

Reply:
No, `/presentation`

Reply:
We could leave /presentation be in GitHub Pages, while the main app is in CloudFlare, since they are not related logically

Reply:
No, we cannot because the repo's name is **cast**, so it will be `/cast/presentation`. The URL https://aabounegm.github.io/presentation would refer to the (non-existing) repo called **presentation**.

Reply:
issue (blocking): this should be `/cast/presentation` then, as @aabounegm mentioned above

Reply:
Or actually, no, never mind, it shouldn't

Reply:
why? Since the main app will be on CloudFlare Pages, why not leave it as just `/cast/` here?